stock_package_info/migrations/10.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+table_renames = [
+    ('rel_package_weight_lot',
+     'stock_picking_package_weight_lot_stock_production_lot'),
+    ('stock_picking_packages_info', 'stock_picking_package_info_ids'),
+]
+
+
+def _rename_module(env):
+    """The old name in v8 was stock_picking_package_info"""
+    openupgrade.update_module_names(
```

Comment:
This is not going to work at this level, as the module is never going to be updated. We should do it outside (ideally on OpenUpgrade's base module), but it can be through a custom one.